### DISCUSSION TITLE
Tuxemon really became awesome recently!

### DISCUSSION THREAD

**Discussion by ultidonki** [Tuxemon/Tuxemon#2312] [General]
I'm currently playing through tuxemon from the beginning, trying to report as many bugs or things that could be improved as possible, but I noticed that tuxemon has become REALLY awesome since the last time I played!

So, I just want to thank everyone for adding the following:
- Being able to view Tuxemon stats/info
- Being able to use the Tuxemon Encyclopedia
- Being able to view what moves do and how powerful they are, rather than randomly guessing
- Battles having status effects and icons
- Night/Day cycle changing lighting and BG's in battle
- Money, being able to buy stuff and Health centre's costing money (well, being added to a bill you don't have to pay off immediately!)
- Being able to store money in a bank
- Being able to view the World Map
- Getting sent back to a Health Center when all your tuxemon are defeated
- A load of other small minigames and interactions you can do in each of the villages

And probably a load of other stuff I haven't noticed yet :smile:

**Comment by Davidliang11292** [comment]
Totally agree! I have to say I like Tuxemon more than Pokemon right now because it belongs to everyone and not just a few companies (which is why I like FOSS software) and I'm really happy to watch Tuxemon get better!
